Remotion/Core/UnitTests/Reflection/ContextAwareTypeUtilityTest.cs

```diff
@@ -122,23 +122,16 @@ public void GetTypeResolutionService_Cached ()
       Assert.That(defaultService, Is.SameAs(defaultService2));
     }
 
-#if NETFRAMEWORK
     [Test]
     public void GetTypeDiscoveryService_WithCustomImplementation_DoesNotThrowOnInitialization ()
     {
-      var relativePath = @"Reflection\TestDomain\ContextAwareTypeUtilityTest\app.config";
-      var fullPath = Path.Combine(TestContext.CurrentContext.TestDirectory, relativePath);
-      Assert.That(File.Exists(fullPath));
+      var typeDiscoveryConfiguration = new TypeDiscoveryConfiguration();
+      typeDiscoveryConfiguration.Mode = TypeDiscoveryMode.CustomTypeDiscoveryService;
+      typeDiscoveryConfiguration.CustomTypeDiscoveryService.Type = typeof(FakeTypeDiscoveryService);
```

Comment:
This test ensures that RM-8047 works correctly (see also RM-8064). We need to improve the naming. It is mandatory that we use the config-files, not intialization by assignment because the type name must be deserialized. The entire setup runs in an isolated AppDomain to ensure the statics are unintialized when things start.

Reply:
Note: if the loadcontext works to re-initialize the statics, loadcontext is still a valid approach